### DISCUSSION TITLE
Is it possible to permanently hide the Autolinks menu from the commit detail view?

### DISCUSSION THREAD

**Discussion by JakubFranek** [gitkraken/vscode-gitlens#2365] [Q&A] [answered]
Hello

I would like to know if it is possible to permanently remove the Autolinks menu from the commit detail view as I do not use this feature and it takes up space.

I am talking about the central part in the following screenshot:
![obrazek](https://user-images.githubusercontent.com/33395319/203384049-6850c8e6-96b0-4298-9f94-19d540f22ca4.png)

I tried disabling all the gitlens.autolinks checkboxes I could find in the settings and restart VSCode but it did not help. Did not manage to Google the answer either.

Thanks!

**Comment by d13** [comment, marked as answer]
Thanks for the question! We do not have an option to hide that section. I'm going to create a feature request for this.

**Reply by JakubFranek** [reply]
Thanks for the quick response, looking forward to the implementation :)
